app/assets/javascripts/ui_models/application.ts

```diff
@@ -257,4 +251,15 @@ export class WebApplication extends SNApplication {
     )(scope);
     angular.element(document.body).append(el);
   }
+
+  public openAccountSwitcher() {
+    const scope = this.scope!.$new(true) as Partial<AccountSwitcherScope>;
+    scope.application = this;
+    const el = this.$compile!(
+      "<account-switcher application='application' "
+      + "class='sk-modal'></account-switcher>"
+    )(scope as any);
```

Comment:
Is there no way to create and insert an angular view other than to compile a string at runtime? 😱

Reply:
The only other way I know of is to insert it in a template with "ng-if" and have that ng-if condition change to true when you want to show it. But I think I would prefer this dynamic way more.